mautic/mautic
# form conditions fields

Documentation PR: https://github.com/mautic/mautic-documentation/pull/208

~⚠️ This PR is blocked by #9761 which needs to be merged before this PR.~

Clone of https://github.com/mautic/mautic/pull/8660 for M3

Closes #8595 

A new feature was requested for combination of Country and conditional State field. The goal is to show only the states for the selected country.

### Steps to test
- Follow https://github.com/mautic/mautic/pull/8660

### To test the Company-state field feature:
1. Create a new form
2. Create a country field
3. Create a new conditional select field for the country field. Make it visible for any country value and map it to the State contact field. The choices should be used from the contact field as well. Do not create custom ones.
4. Save the form.
5. Go to the form preview.


<a href="https://gitpod.io/#https://github.com/mautic/mautic/pull/8794"><img src="https://gitpod.io/button/open-in-gitpod.svg"/></a>
